arm/Microsoft.MachineLearningServices/workspaces/deploy.bicep

```
@@ -153,6 +171,26 @@ resource workspace 'Microsoft.MachineLearningServices/workspaces@2021-04-01' = {
   }
 }
 
+module workspace_computes 'computes/deploy.bicep' = [for compute in computes: {
+  name: '${workspace.name}-${compute.name}-compute'
+  params: {
+    machineLearningWorkspaceName: workspace.name
+    name: compute.name
+    location: location
+    sku: sku
```

Do we want this to rely on the values from within the compute object that you iterate through or should it be inherited from the parent?

```suggestion
    sku: compute.sku
```

---

@MariusStorhaug Updated.